test/e2e/scheduling/taints.go

```diff
@@ -166,10 +166,13 @@ var _ = SIGDescribe("NoExecuteTaintManager Single Pod [Serial]", func() {
 		framework.ExpectNoError(err)
 	})
 
-	// 1. Run a pod
-	// 2. Taint the node running this pod with a no-execute taint
-	// 3. See if pod will get evicted
-	It("evicts pods from tainted nodes", func() {
+	/*
+		Release : v1.15
+		Testname: Taint, Pod Eviction on no-execute
+		Description: Create a Pod and wait for it to be scheduled onto a Node. Taint the concerned Node
+						with a no-execute taint and wait for the Pod to be evicted. The test MUST pass.
```

Comment:
please describe the behavior in terms of MUST language, not the test.. ie: the post MUST be evicted

Reply:
If this test is worth promoting, why not the below test case that uses tolerations?

Reply:
I agree, this test also needs to have atleast 2 additional cases that I can think of

- Add a toleration for `No-Execute` taint and wait if it tolerates.
- Add `tolerationSeconds` with some value like `120` and check if we are waiting for those seconds before evicting.

@globervinodhn - Will you be able to update this PR with those changes?

Reply:
That's a lot of time to wait for eviction. I'm not asking so much for writing more test cases, just that if we're promoting this test case, there appear to be more within this file that are eligible.

Reply:
https://github.com/kubernetes/kubernetes/pull/77331#issuecomment-491965511